register/validators.py

```diff
@@ -0,0 +1,17 @@
+from django.core.exceptions import ValidationError
+
```

Comment:
import models here

Reply:
it is imported with in the definition because otherwise there is an import error

Reply:
I meant instead of writing the import statement in the methods. Write it over here.

Reply:
if you write it up top then you get an import error for some reason but importing in the method makes it work.